cmd/validator/dynamic_processor.go

```diff
@@ -0,0 +1,275 @@
+package main
```

Comment:
This file has a comparison function that follows the same algorithm as the jq normalization in the bash script. The output it provides on a mismatch isn't as good as the other algorithm, so this is mainly for reference....